src/components/departureTiles/departureTile.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { getIcon, groupBy, isVisible } from '../../utils'
+
+const DepartureTile = ({ stopPlace, routes, hiddenRoutes }) => {
+    const { departures, name, id } = stopPlace
+    const groupedDepartures = groupBy(departures, 'route')
+
+    if (isVisible(groupedDepartures, hiddenRoutes)) {
```

Comment:
For å redusere indentering, så ville eg snudd if-sjekken. Altså slik: 

```
 if (!isVisible(groupedDepartures, hiddenRoutes)) {
    return null
}

return (
    <div className="tile-container" key={id}>
...
```

Reply:
![](https://media.giphy.com/media/8i479V7gzkUdG/giphy-downsized.gif)